### PR TITLE
[SeaTunnel #780][issue] Add umbrella issue template

### LINKED ISSUE #780: [Feature][issue] Add umbrella issue template
### Search before asking

- [X] I had searched in the [feature](https://github.com/InterestingLab/seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

Add umbrella issue template

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by leo65535**
Fixed #780 

## Purpose of this pull request

Add umbrella issue template.

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] Change does not need document change, or I will submit document change to https://github.com/InterestingLab/seatunnel-docs later

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/umbrella.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    validations:
+      required: true
+
+  - type: checkboxes
+    attributes:
+      label: Are you willing to submit PR?
+      description: >
+        A pull request is optional, but we are glad to help you in the contribution process
+        especially if you already have a grip on how to implement the new feature.
+        Kyuubi is a community-driven project and we love to bring new contributors in.
```
Comment (wuchunfu) [inline .github/ISSUE_TEMPLATE/umbrella.yml:74]: Please change `Kyuubi` to `seatunnel`

File: .github/ISSUE_TEMPLATE/umbrella.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      description: >
+        This is absolutely not required, but we are happy to guide you in the contribution process
+        especially if you already have a good understanding of how to implement the fix.
+        seatunnel is a totally community-driven project and we love to bring new contributors in.
+        options:
+        - label: Yes I am willing to submit a PR!
+
+  - type: markdown
+    attributes:
+      value: "Thanks for taking the time to propose an umbrella issue!"
```
Comment (CalvinKirs) [inline .github/ISSUE_TEMPLATE/umbrella.yml:80]: https://gh-community.github.io/issue-template-feedback/structured/#body1-required-attribute-key-value-is-missing
image
hi, there is a small error

**Review by CalvinKirs — COMMENTED** [review]
body[4]: Required attribute key options is missing. Learn more.

File: .github/ISSUE_TEMPLATE/umbrella.yml (hunk 2 above)
Comment (leo65535) [reply .github/ISSUE_TEMPLATE/umbrella.yml:80]: Thanks.

**Comment by leo65535** [comment]
@CalvinKirs thanks for your review, updated.

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**